build/yamls/base/agent.yml

```diff
@@ -113,6 +113,8 @@ spec:
             mountPath: /etc/antrea/antrea-agent.conf
             subPath: antrea-agent.conf
             readOnly: true
+          - name: antrea-ca
+            mountPath: /var/run/antrea/secrets
```

Comment:
is using the same path as to mount the actual secret in the controller a good idea? I think we can use the name of the config map (`antrea-ca`) instead of `secrets`.